src/cswinrt/strings/additions/Windows.Storage.Streams/IBufferByteAccess.cs

```diff
@@ -18,6 +18,20 @@ namespace ABI.Windows.Storage.Streams
     using global::System.Runtime.InteropServices;
     using global::System.ComponentModel;
 
+    internal static class IBufferByteAccessMethods
+    {
+        public static unsafe IntPtr get_Buffer(IObjectReference _obj)
+        {
+            var ThisPtr = _obj.ThisPtr;
+
+            IntPtr __retval = default;
+            global::WinRT.ExceptionHelpers.ThrowExceptionForHR((*(delegate* unmanaged[Stdcall]<IntPtr, IntPtr*, int>**)ThisPtr)[6](ThisPtr, &__retval));
```

Comment:
Is the index here right?  The interface looks to implement IUnknown.

https://learn.microsoft.com/en-us/windows/win32/api/robuffer/ns-robuffer-ibufferbyteaccess